symengine/expression.h

```diff
@@ -173,11 +173,6 @@ inline Expression expand(const Expression &arg)
     return expand(arg.get_basic());
 }
 
-inline bool unified_eq(const Expression &a, const Expression &b)
```

Comment:
Yes, I'm sure. You can just check on master, try calling `unified_eq` on two `umap_basic_num`.